workflows/variantcall.nf

```diff
@@ -41,37 +39,23 @@ workflow VARIANTCALL {
     //     params.chain_checksum
     // )
 
-    // SanityCheck
-    sanityCheck(
+    // Prepare metadata, adapted from SanityCheck
+    PREP_META(
         file(params.experiment_info_tsv),
```

Comment:
Include a `checkIfExists` for files:
https://www.nextflow.io/docs/latest/channel.html#frompath